server/controllers/dashboardController.js

```diff
@@ -60,15 +22,37 @@ const paginationData = async (req, res) => {
 
   const { dateFrom, dateTo } = isValidStartDate(date);
 
-  const allData = await UpsellingPartnerQuery('onboarding', offset, limit, dateFrom, dateTo);
+  const allData = await upsellingPartnerQuery('onboarding', offset, limit, dateFrom, dateTo);
```

Comment:
Avoid using hardcoded (string) values; 
See comments on the `upsellingPartnerQuery`

Reply:
What if that function will be reused??

Reply:
You can then pass in a different `argument`. 
Point is, you're using a hardcoded string, rather, put the value in an externally accessible variable so that the function can easily be reused by changing the value of that variable